flake.nix

```diff
@@ -321,6 +321,7 @@
           packages =
             [
               alejandra
+              buildkite-test-collector-rust
```

Comment:
We'll get newer version that auto-update after https://github.com/NixOS/nixpkgs/pull/294343 is unblocked and then merged.